grommet-leaflet-core/src/markers/Popup.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Box, Grommet } from 'grommet';
+import { hpe } from 'grommet-theme-hpe';
+import '../grommet-leaflet-reset.css';
+
+const Popup = ({ ...rest }) => {
+  return (
+    <Grommet theme={hpe} background="transparent">
```

Comment:
I want to continue to explore how we can avoid these additional Grommet instances. Feels excessive.

Reply:
That would be nice. At the moment, without the Grommet tag, the components only get the "base.js" styling since we are rendering via ReactDOMServer. If we can find a way around that down the line that'd be best.